web/viewer.html

```diff
@@ -98,12 +98,8 @@
 
       <div id="mainContainer">
         <div class="findbar hidden doorHanger" id="findbar">
-          <div id="findbarLabelContainer">
```

Comment:
Is it too much of a hassle to move the "placeholder" commit such that it's the first one in the series, since then you'd not need to first add the `findbarLabelContainer` and the immediately remove it in the next commit thus reducing the churn slightly in this code?